tests/utils/shippable/sanity.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+
+set -o pipefail -eux
+
+# shellcheck disable=SC2086
+ansible-test sanity --color -v --junit ${COVERAGE:+"$COVERAGE"} ${CHANGED:+"$CHANGED"} \
+    --docker \
```

Comment:
That trailing backslash can be harmful:
```suggestion
    --docker
```